jyulb/internal/common/domain.pyx

```diff
@@ -0,0 +1,195 @@
+import cython
+import numpy as np
+cimport numpy as np
```

Comment:
These two imports can be confused, I would rename the seconds as `cimport numpy as cnp`